### PR TITLE
fix(readme): Add links for `aqueductcore-dev` images on Dockerhub to README.md

### PR DESCRIPTION
**Pull request by samiralavi**
This PR adds the information about the name of the docker image used for development on the README.md file. Later, once we have a developer section in the docs, this could be added there too, but at this time the docs only contain release information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,37 +76,40 @@ services:
...
+   start the server.
 
 3. Start your browser and point it to `https://localhost`.
 
 For more information please check the [documentation](https://aqueducthub.github.io/aqueductcore/).
 
+#### Development Docker Images
+
+You can find development docker images from the head of the main branch on DockerHub, `aqueducthub/aqueductcore-dev:latest`. The images are built after each commit or pull request merge to the `main` branch.
+
```
Comment (str-anger) [inline README.md:90]: From my experience it is always useful to have the TL;DR version of documentation, which summarises all actions into a single script. Probably something like this:

Default way to install acqueductcore on your machine is:
```
docker pull aqueducthub/aqueductcore-dev:latest
docker run aqueducthub/aqueductcore-dev:latest
```

**Review by str-anger — APPROVED** [review]
I have a non-blocking suggestion, but it has to validated, mu docker knowledge is an a baby level =)

File: README.md (hunk 1 above)
Comment (samiralavi) [reply README.md:90]: The `docker pull` command is easy but the `docker run` command can be messy. Environments should be passed as arguments along with port configuration. That is the reason we prefer docker-compose which defines the whole config in a `yaml` file.

Comment (str-anger) [reply README.md:90]: But still, is there a "one, two, three" receipt, which used may follow, to start an app? Currently it is scattered through the text.

**Review by safeamiiir — APPROVED** [review]
Happy to have this information added into the README!

File: README.md (hunk 1 above)
Comment (samiralavi) [reply README.md:90]: I see your point. This looks to me you are proposing a revision of our README and docs file. I suggest you create a PR with your requested changes then.

**Pull request merged**